src/View/Shortcodes/EmbedShortcodeProvider.php

```diff
@@ -178,8 +179,37 @@ protected static function linkEmbed($arguments, $href, $title)
      */
     protected static function photoEmbed($arguments, $src)
     {
-        $arguments['src'] = $src;
-        unset($arguments['url']);
-        return HTML::createTag('img', $arguments);
+        $data = [
+            'Arguments' => $arguments,
+            'Attributes' => static::buildAttributeListFromArguments($arguments, ['url']),
+            'Src' => $src
+        ];
+
+        return ArrayData::create($data)->renderWith(self::class . '_photo')->forTemplate();
+    }
+
+    /**
+     * Build a list of HTML attributes from embed arguments - used to preserve backward compatibility
+     *
+     * @deprecated 5.0 Use {$Arguments.name} directly in shortcode templates to access argument values
```

Comment:
```suggestion
     * @deprecated 4.5.0 Use {$Arguments.name} directly in shortcode templates to access argument values
```

Assuming you want this deprecated _from_ 4.5